src/Chirp.Core/AuthorRepository.cs

```diff
@@ -0,0 +1,57 @@
+using Chirp.Core.DataModel;
+
+namespace Chirp.Core;
+
+public interface IAuthorRepository
+{
+    public Author? GetAuthor(string? authorName = null, string? authorEmail = null);

```

Comment:
I believe this should be two signatures, as this poses the questions:
1. Can an author be retrieved by both name and email at the same time?
2. Can an author by retrieved without any parameters?

This signature allows for both of these.